### PR TITLE
shippingservice docs

### PR DESCRIPTION
**Pull request by GaryPWhite**
## Changes

Docs for shippingservice -- matched with other docs contributions for similar services.

### CODE REVIEW

File: docs/services/shippingservice.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+Spans and other metrics are created in this example throughout `tokio` async runtimes
+found within [`tonic` server functions](https://github.com/hyperium/tonic/blob/master/examples/helloworld-tutorial.md#writing-our-server).
+Be mindful of async runtime, [context guards](https://docs.rs/opentelemetry/latest/opentelemetry/struct.ContextGuard.html),
+and inability to [move and clone `spans`](https://stackoverflow.com/questions/25649423/sending-trait-objects-between-threads-in-rust)
+when replicating from these samples.
+
+### Adding GRPC/HTTP instrumentation
+
+This service receives HTTP requests, which are instrumented in the middleware.
```
Comment (julianocosta89) [inline docs/services/shippingservice.md:53]:
```suggestion
This service receives gRPC requests
```

Is thie following true for the `shippingservice`?
```
 which are instrumented in the middleware.
```
I know that this is valid for PHP, but not sure if that applies to Rust

File: docs/services/shippingservice.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+The middleware starts root span, and can pass reference in the same thread down
```
Comment (julianocosta89) [inline docs/services/shippingservice.md:55]:
```suggestion
The root span is started and passed down as reference in the same thread
```

**Review by julianocosta89 — CHANGES_REQUESTED** [review]
I've noted down just some small adjusts

File: docs/services/shippingservice.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
+# Shipping Service
+
+This service is responsible for providing shipping information including pricing
+and tracking information, when requested from Checkout Service.
+
+Shipping service is built primarily with Tonic, Reqwest, and opentelemtry. Other
```
Comment (mviitane) [inline docs/services/shippingservice.md:6]: opentelemtry -> OpenTelemetry libraries/components?

File: docs/services/shippingservice.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        span.set_attribute(KeyValue::new("app.shipping.cost.total", format!("{}", q)));
+        q
+    }))
+```
+
+Note that the span cannot be used after calling an async method in the
+originating function.
+
+You may also notice the `attributes` set on the span in this example, and
+`events` propogated similarly. With any valid `span` pointer (attatched to
```
Comment (mviitane) [inline docs/services/shippingservice.md:92]: ->attached

File: docs/services/shippingservice.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+context) the [opentelemetry api](https://docs.rs/opentelemetry/0.17.0/opentelemetry/trace/struct.SpanRef.html)
```
Comment (mviitane) [inline docs/services/shippingservice.md:93]: opentelemetry api -> OpenTelemetry API

File: docs/services/shippingservice.md (hunk 3 above)
Comment (GaryPWhite) [reply docs/services/shippingservice.md:6]: fixed in commit

File: docs/services/shippingservice.md (hunk 4 above)
Comment (GaryPWhite) [reply docs/services/shippingservice.md:92]: yeesh typo there thanks!

File: docs/services/shippingservice.md (hunk 5 above)
Comment (GaryPWhite) [reply docs/services/shippingservice.md:93]: got it

File: docs/services/shippingservice.md (hunk 1 above)
Comment (GaryPWhite) [reply docs/services/shippingservice.md:53]: I would consider [tonic](https://github.com/hyperium/tonic) in this case to be acting as middleware.

Comment (julianocosta89) [reply docs/services/shippingservice.md:53]: But it is gRPC right?

Comment (julianocosta89) [reply docs/services/shippingservice.md:53]: It receives gRPC and sends HTTP

**Comment by julianocosta89** [comment]
@GaryPWhite it receives gRPC and sends HTTP right?

**Comment by GaryPWhite** [comment]
@julianocosta89 it does! It receives grpc and send HTTP

**Comment by GaryPWhite** [comment]
ah, I see. SOrry bout that. Lemme update and post again.

**Review by julianocosta89 — APPROVED** [review]
Great! This is good to go!

**Pull request merged**